llama_stack/providers/remote/inference/ollama/config.py

```diff
@@ -6,13 +6,13 @@
 
 from typing import Any, Dict
 
-from pydantic import BaseModel
+from pydantic import BaseModel, Field
 
 DEFAULT_OLLAMA_URL = "http://localhost:11434"
 
 
 class OllamaImplConfig(BaseModel):
-    url: str = DEFAULT_OLLAMA_URL
+    url: str = Field(DEFAULT_OLLAMA_URL, json_schema_extra={"user_field": True})
```

Comment:
why is this added only here? is this annotation not relevant to other places?

Reply:
apologies, leftover, thanks for catching this! Removed